packages/plugin-compat/extra/typescript/gen-typescript-patch.js

```diff
@@ -80,10 +80,17 @@ const SLICES = [
   },
   // https://github.com/merceyz/TypeScript/tree/merceyz/pnp-4.5
   {
-    from: `79fe5d206f89e5f1585807bdcd7feeb567a69646`,
-    to: `fff89a12766fa485e35b1c65d8631d109c4e2c0e`,
-    onto: `7893f689fedcfe28a499d119d34844840bce9160`,
-    range: `>=4.5.2`,
+    from: `9232978f8e54f073b5451d0bf2737d42a0fd672f`,
+    to: `3a2388d39d41d000b5c5f9bcd48096b39fcedf8f`,
+    onto: `55e13e9115b3cc5458d76c39da1211dc28d7b51f`,
```

Comment:
Why did you need to change the from/to/onto?

Reply:
There has been a few releases for 4.5 since we last generated the patch so I updated my branch, I'll revert it since it isn't strictly needed.